in-toto-spec.md

```diff
@@ -1158,13 +1245,18 @@ metadata:
       "stdin": "",
       "stdout": "foo.py",
       "return-value": "0"
+    },
+    "environment": {
+      "variables": [""],
+      "filesystem" : "",
+      "workdir": ""
     }
   },
   "signatures" : [
     { "keyid" : "<BOBS_KEYID>",
       "method" : "ed25519",
       "sig" :
-          "ae3aee92ea33a8f461f736a698e082e12c300dfe5022a06c7a6c2a6a93a9f5771eb2e5ce0c93dd580bebc2080d10894623cfd6eaedf1d5aa394df84890d7ace3"
+          "ae3aee92ea33a8f461f736a698e082e12c300dfe5022a06c7a6c2a6a"
```

Comment:
What's the reason you removed part of the signature?
(In Alice's link above you left the signature as it was)

Reply:
I mostly did this for readability. I don't know if having a 30-char length hex string adds more information than a shorter one. On the examples outside of the spec I ellipsized it, would this help here?